Pbi 254312 projectile rebranding (#698)
osisoft/OCS-Docs: content/add-organize-data/organize-data/metadata-rules/metadata-rules-procedure.md

1. In the left pane, select **Data Management** > **Sequential Data Store**.

> 2. On the `Sequential Data Store` page, click **Streams**.

Reviewer: ```suggestion

1. On the `Sequential Data Store` page, click **Streams**.
```

> 3. Click the **Change Namespace** button in the upper-right toolbar, then select the namespace in which you want to create the stream.

Reviewer: ```suggestion

1. Click the **Change Namespace** button in the upper-right toolbar, then select the namespace in which you want to create the stream.
```

> 4. Expand the `Streams` pane if necessary to copy the full name of the stream that will be the basis for your metadata rule.

Reviewer: ```suggestion

1. Expand the `Streams` pane if necessary to copy the full name of the stream that will be the basis for your metadata rule.
```

> 5. In the left pane, click **Data Management** > **Metadata Management**.

Reviewer: ```suggestion

1. In the left pane, click **Data Management** > **Metadata Management**.
```

> 11. For each delimited section of the stream name you select, use the drop-down list to assign a metadata type: Metadata, String Literal, or Wildcard. See the table following this procedure for a description of these metadata types. 

Reviewer: ```suggestion
1. For each delimited section of the stream name you select, use the drop-down list to assign a metadata type: Metadata, String Literal, or Wildcard. See the table following this procedure for a description of these metadata types. 
```

> 12. After entering all metadata type entries, click **Next**.

Reviewer: ```suggestion
1. After entering all metadata type entries, click **Next**.
```

    **Note**: If you select metadata type **Metadata**, you must also enter a metadata key. A metadata key is the key portion of a key-value pair, typically the type of data provided, such as Measurement. This is helpful when streams that match the rule provide different types of data. Entries in the **Map To** portion of mapping represent the value part of the key-value pair. For example, Power would be the defined key, while different types of data could be Demand, TOTAL/Electricity, or Metering. The mapping entries specify the labels you want to display for each of those values.
    

> 13. In the `Define Mappings` page, choose one of the following options:

Reviewer: ```suggestion
1. In the `Define Mappings` page, choose one of the following options:
```

     - To display the raw stream data for the specified metadata key, select **Copy Values** under **Mapping Type**.
     - To assign a label to data values, select **Map Values** and click **Generate Mappings**.

> 14. If you selected **Map Values**, then do the following: In the **Map To** field, enter the label you want displayed for each defined metadata key, and click **Next**.

Reviewer: ```suggestion

1. If you selected **Map Values**, then do the following: In the **Map To** field, enter the label you want to display for each defined metadata key, and click **Next**.
```

> 15. Optional: Click **Add Mapping** to define a mapping for any other stream name part.

Reviewer: ```suggestion

1. Optional: Click **Add Mapping** to define a mapping for any other stream name part.
```

> 16. Click **Next**. All matching streams for the rule are displayed.
> 17. Enter the metadata rule **Name** and **Description** in the `Preview and Run` page.
> 18. Click **Save & Execute**.

Reviewer: ```suggestion

1. Click **Next**. All matching streams for the rule are displayed.

1. Enter the metadata rule **Name** and **Description** in the `Preview and Run` page.

1. Click **Save & Execute**.
```
